src/plugins/data/public/types.ts

```diff
@@ -42,7 +42,7 @@ import { IndexPatternsContract } from './index_patterns';
 import { IndexPatternSelectProps, StatefulSearchBarProps } from './ui';
 import { UsageCollectionSetup } from '../../usage_collection/public';
 import { DataSourceStart } from './data_sources/datasource_services/types';
-
+import { ObservabilitySetup } from '../../../../plugins/dashboards-observability/public/types';
```

Comment:
This import breaks when the observability plugin isn't installed right? Isn't there a better way to check if the plugin is installed? The data source selector uses an API to do it I believe

Reply:
Updated it using application.application$ service to determine if observability plugin is intalled.